src/css/custom.scss

```diff
@@ -90,55 +90,8 @@ select {
     left: 0;
 }
 
-.card {
-    border: 0 !important;
-}
-
-.disabled-card {
-    .mod-logo, .card-title, .card-content {
-        opacity: 0.6;
-    }
-}
-
-.card-footer-borderless, .card-footer-borderless .card-footer-item {
-    border: none !important;
-    // these two lines allow the hover effect's pseudo-element to position itself behind the element's text, with a bit of a margin
-    position: relative;
-    z-index: 1;
-}
-
-.card-footer-item:not(.non-selectable) {
-
-    &::before {
-        content: "";
-        display: block;
-        position: absolute;
-        top: 10px;
-        left: 10px;
-        width: calc(100% - 20px);
-        height: calc(100% - 20px);
-        z-index: -1;
-        background-color: transparent;
-        border: 1px solid $scheme-border-secondary;
-        border-radius: 4px;
-    }
-
-    &:hover, &:active, &:focus {
-        color: #ffffff;
-        &::before {
-            background-color: $scheme-primary;
-            border: 0;
-        }
-    }
-}
-
-.card-byline, .card-timestamp {
-    color:$scheme-text-secondary;
-}
-
-.card-timestamp strong {
-    color: inherit;
-}
+@import "components/Card";
```

Comment:
Minor nitpick but SCSS filenames are currently lowercase

Reply:
Ah, yeah, that's my bad, I'm used to my own way of doing things which is camelCase for "general" files and PascalCase for singular components. I'm guessing you'll prefer kebab-case? I'll remember to use that instead